Dockerfile

```diff
@@ -3,7 +3,6 @@ ARG TARGETARCH
 # Build the manager binary
 FROM docker.io/library/golang:1.24 as builder
 
-ARG TARGETARCH
```

Comment:
this must not be removed, it's used in the line below:
```
RUN CGO_ENABLED=0 GOARCH=$TARGETARCH go build
```

Reply:
hmm that I don't remember why will revert as well and if needed I will keep it locally

Reply:
I don't think that option is there for `make docker-generate`